packs/data/spells.db/stumbling-curse.json

```
@@ -26,7 +26,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>With a twist of a finger, you send a creature careening on a haphazard trajectory. The target must attempt a Reflex save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target takes a -2 status penalty to Athletics and Acrobatics checks.</p>\n<p><strong>Failure</strong> As success, and each time the target Strides, it uses the first 5 feet of movement of the Stride to move 5 feet in a random direction (this is forced movement). This has no effect on a creature with only 5 feet of movement.</p>\n<p><strong>Critical Failure</strong> As failure, except you control the direction of the target's first 5 feet of movement of the Stride.</p>"
+            "value": "<p>With a twist of a finger, you send a creature careening on a haphazard trajectory. The target must attempt a Reflex save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target takes a -2 status penalty to Athletics and Acrobatics checks.</p>\n<p><strong>Failure</strong> As success, and each time the target @Compendium[pf2e.actionspf2e.Stride]{Strides}, it uses the first 5 feet of movement of the Stride to move 5 feet in a random direction (this is forced movement). This has no effect on a creature with only 5 feet of movement.</p>\n<p><strong>Critical Failure</strong> As failure, except you control the direction of the target's first 5 feet of movement of the Stride.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.ViBlOrd6hno3DiPP]{Spell Effect: Stumbling Curse}</p>"
```

We don't normally link to the super basic actions like Stride
```suggestion
            "value": "<p>With a twist of a finger, you send a creature careening on a haphazard trajectory. The target must attempt a Reflex save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target takes a -2 status penalty to Athletics and Acrobatics checks.</p>\n<p><strong>Failure</strong> As success, and each time the target Strides, it uses the first 5 feet of movement of the Stride to move 5 feet in a random direction (this is forced movement). This has no effect on a creature with only 5 feet of movement.</p>\n<p><strong>Critical Failure</strong> As failure, except you control the direction of the target's first 5 feet of movement of the Stride.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.ViBlOrd6hno3DiPP]{Spell Effect: Stumbling Curse}</p>"
```